docs/content/documentation/modules/porous_flow/capillary_pressure.md

```diff
@@ -1,6 +1,30 @@
 #Capillary pressure
 
-A number of capillary pressure curves are available in Porous Flow.
+The capillary pressure is the pressure difference between two fluid phases in a porous
+medium that arises due to the interfacial tension between the fluid phases and the surface
+tension between fluids and the porous medium. Capillary pressure, $P_c$, is commonly
+defined as \citep{Bear1972}
```

Comment:
It seems to be, but you are correct, it should be all lowercase. I'll fix this up on my next edit of the documentation